lib/react_client.dart

```diff
@@ -400,9 +400,10 @@ class ReactDomComponentFactoryProxy extends ReactComponentFactoryProxy {
     var children = _convertArgsToChildren(childrenArgs);
     children = listifyChildren(children);
 
-    convertProps(props);
+    var convertibleProps = {}..addAll(props);
```

Comment:
This could use a comment: 
```dart
// We can't mutate the original since we can't be certain that the value of the
// the converted event handler will be compatible with the Map's type parameters.
```